.gitignore

```diff
@@ -55,4 +55,7 @@ notes.txt
 tsout/
 
 # Test results
-test-results/
\ No newline at end of file
+test-results/
+
+# Test @metamask package publishing
+.npmrc
```

Comment:
This is a temporary change and I can bring this out into a separate PR. For now, I've explained why this is necessary in the PR description.

Reply:
Well, I guess if we have the CircleCI config in this branch, we can have this here too since they're related.